clang/tools/dpct/bash-autocomplete.sh

```diff
@@ -0,0 +1,66 @@
+# Please add "source /path/to/bash-autocomplete.sh" to your .bashrc to use this.
```

Comment:
Please consider to install this script to install folder when run "ninja install-dpct"